cli/plasmo/templates/static/svelte3/content-script-ui-mount.ts

```diff
@@ -16,31 +18,29 @@ const createMountContainer = () => {
     left: 0px;
   `
 
-  if (typeof Mount.getMountPoint !== "function") {
-    return container
-  }
+  if (typeof Mount.getOverlayAnchor === "function") {
+    const updatePosition = async () => {
+      const anchor = (await Mount.getOverlayAnchor()) as HTMLElement
 
-  const updatePosition = async () => {
-    const anchor = (await Mount.getMountPoint()) as HTMLElement
+      const rect = anchor?.getBoundingClientRect()
 
-    const rect = anchor?.getBoundingClientRect()
+      if (!rect) {
+        return
+      }
 
-    if (!rect) {
-      console.error("getMountPoint is not returning a valid HTMLElement")
-      return
-    }
+      const pos = {
+        left: rect.left + window.scrollX,
+        top: rect.top + window.scrollY
+      }
 
-    const pos = {
-      left: rect.left + window.scrollX,
-      top: rect.top + window.scrollY
+      container.style.top = `${pos.top}px`
+      container.style.left = `${pos.left}px`
     }
 
-    container.style.top = `${pos.top}px`
-    container.style.left = `${pos.left}px`
+    updatePosition()
+    window.addEventListener("scroll", updatePosition)
```

Comment:
It'd be nice if we also offered our users an ability to manually trigger an update to the position (i.e. something's changed on the page, and now the position is no longer in the right place)

Reply:
Added a `watchOverlayAnchor` callback for user, passing in the updatePosition for them. I think that should work